### ISSUE TITLE
Problem with alwaysSendToDefaultTracker: false & ReactGA.addTrackers

### ISSUE THREAD

**Issue by zehawki** [react-ga/react-ga#462]
TL;DR alwaysSendToDefaultTracker setting to false does not take effect when using ReactGA.addTrackers.

--

All testing done with Google Analytics Debugger. Looking at console, I see the following behavior:

After initialization via ReactGA.initialize, I make a call to ReactGA.pageview(page, ['Client']):
_Running command: ga("create", "UA-181101771-1", "auto")
VM436 analytics.js:25 Running command: ga("send", {hitType: "pageview", page: "/topics/health-wellness"})
VM436 analytics.js:25 Running command: ga("Client.send", {hitType: "pageview", page: "/topics/health-wellness"})
VM436 analytics.js:25 Command ignored. Unknown target: undefined_

This is fine. At this point, 2 calls are made as expected. The 2nd target has not been added and is corrected ignored by GA. All good so far.

Next "Client" tracker is added via ReactGA.addTrackers with alwaysSendToDefaultTracker: false, and then I make a call to ReactGA.pageview(page, ['Client']):
_Executing Google Analytics commands.
VM436 analytics.js:25 Running command: ga("create", "UA-131660881-1", {name: "Client"})
VM436 analytics.js:25 Creating new tracker: Client
VM436 analytics.js:25 Running command: ga("send", {hitType: "pageview", page: "/topics/health-wellness"})
VM436 analytics.js:25 Running command: ga("Client.send", {hitType: "pageview", page: "/topics/health-wellness"})_

Here, I would expect that the event would not be send to the default tracker. But as one can see above, it IS getting sent.

**Comment by zehawki** [comment]
Possible workaround: Setting alwaysSendToDefaultTracker: false in ReactGA.initialize and again in ReactGA.addTrackers makes this work. 

Oh well, maybe thats how its supposed to be. Would be good to mention in the docs.
